src/components/MultiLink/Legend.vue

```diff
@@ -297,6 +328,19 @@ export default Vue.extend({
       }
     },
 
+    renderScales(barVar: string) {
+      const varMin = this.attributeRanges[barVar].min;
+      const varMax = this.attributeRanges[barVar].max;
+
+      const scale = scaleLinear()
+        .domain([varMin, varMax])
+        .range([this.stickyBarHeight, 0]);
+      const axis = axisRight(scale).ticks(4, 's');
+
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      select(`#node_${barVar}_scale`).call((axis as any));
```

Comment:
Is this due to bad d3 typing?

Reply:
It was. I managed to remove the `disable-next-line` by appending a g before calling the axis function on it in 6825a00